### ISSUE TITLE
Need more specific doc.

### ISSUE THREAD

**Issue by dovahkrid** [Infinical/torrent_streamer#1]
I clone your example (which is not shown in your repo but doc) and it always return the error

`Exception: Initialise torrent streamer before using it!`

Even when I edit the code to

<img width="409" alt="Screen Shot 2021-12-13 at 15 39 45" src="https://user-images.githubusercontent.com/14861163/145779189-37b67dde-c46c-42a7-ba93-38e42ae006fc.png">

It still does not work when run `_startDownload` with the same error.

Looking for the update !

**Comment by QFREY** [comment]
I have the same problem did you figure out a way to make it work yet

**Comment by dovahkrid** [comment]
> I have the same problem did you figure out a way to make it work yet

Nope. After added init(), it worked when add listener. But after that, it still `Exception: Initialise torrent streamer before using it!`

**Comment by QFREY** [comment]
I got it past that but then it gave me another error about a non existing dispose method.

**Comment by dovahkrid** [comment]
> I got it past that but then it gave me another error about a non existing dispose method.

The dispose error is only a warning. After you get through that, you still get the `Exception: Initialise torrent streamer before using it!` again

**Comment by AdvaitKale01** [comment]
Still same error. Seems this package is dead.

**Comment by risfat** [comment]
Any solutions yet ?
